user_agent.go

```diff
@@ -153,7 +153,7 @@ func (p *UserAgent) Parse(ua string) {
 }
 
 // Returns the mozilla version (it's how the User Agent string begins:
-// "Mozilla/5.0 ...", unless we're dealing with Opera, of course).
+// "Mozilla/5.0 ...", unless we're dealing with Opera or Dalvik, of course).
```

Comment:
I thought you said you want it to be Mozilla/5.0, no?
You said: "I'd say just set the Mozilla attribute to 5.0, and that's it."